config/database.cr

```diff
@@ -1,5 +1,19 @@
 database_name = "avram_dev"
 
+# Adding this struct makes everything work
+#
+# Removing it causes:
+#
+# BUG: trying to upcast Dexter::Formatters::JsonLogFormatter <- Dexter::Formatters::BaseLogFormatter (Exception)
```

Comment:
Link to JsonLogFormatter: https://github.com/luckyframework/dexter/blob/master/src/dexter/formatters/json_log_formatter.cr